### PR TITLE
add documentation for switching color themes 

### PR DESCRIPTION
**Pull request by toalba**

### CODE REVIEW

**Review by stevepiercy — DISMISSED** [review]
I approve, but I would like @MrTango or another content expert from the Classic UI team to review.

If we don't hear back within a week, please ping me to remind me, and I'll go ahead and merge this. Thank you!

**Review by petschki — CHANGES_REQUESTED** [review]
Documentation is good, but it's based on the not yet released barceloneta/mockup/staticresources versions introducing Bootstrap 5.3 ... we agreed at the alpine city sprint to release this as minor feature versions (semantic) and ship it with Plone 6.1 but you can use those versions also in Plone 6.0 ... @toalba it would be good to mention this in the docs somewhere.

**Review by MrTango — DISMISSED** [review]
what petschki says ;)

File: docs/classic-ui/theming/color-mode.md [hunk 1]
```diff
@@ -141,8 +141,12 @@ See the following example.
 <form data-bs-theme='light'>
   <div class="mb-3">
     <label for="exampleInputEmail1" class="form-label">Email address</label>
-    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+    <input type="email" class="form-control" id="exampldescribtioneInputEmail1" aria-describedby="emailHelp">
```
Comment (stevepiercy) [inline docs/classic-ui/theming/color-mode.md]: What happened here? Did you mean `exampleDescriptionInputEmail1`?

File: docs/classic-ui/theming/color-mode.md [hunk 2]
```diff
@@ -12,16 +12,16 @@ myst:
 # Color modes
 
 Bootstrap 5.3 has introduced [Color Modes](https://getbootstrap.com/docs/5.3/customize/color-modes/).
-This chapter is a guide for how to implement color themes in Plone 6.
+This chapter is a guide for how to implement color themes in Plone 6.1.
```
Comment (stevepiercy) [inline docs/classic-ui/theming/color-mode.md]:
````suggestion
This chapter is a guide for how to implement color modes in Plone 6.1.

```{versionadded} Plone 6.1
```

````

File: docs/classic-ui/theming/color-mode.md [hunk 3]
```diff
@@ -12,16 +12,16 @@ myst:
 # Color modes
 
 Bootstrap 5.3 has introduced [Color Modes](https://getbootstrap.com/docs/5.3/customize/color-modes/).
```
Comment (stevepiercy) [inline docs/classic-ui/theming/color-mode.md]:
```suggestion
Bootstrap 5.3 has introduced [color modes](https://getbootstrap.com/docs/5.3/customize/color-modes/).
```

File: docs/classic-ui/theming/color-mode.md [hunk 4]
```diff
@@ -141,8 +141,12 @@ See the following example.
...
-    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+    <input type="email" class="form-control" id="exampldescribtioneInputEmail1" aria-describedby="emailHelp">
   </div>
   <button type="submit" class="btn btn-primary">Submit</button>
 </form>
 ```
+
+## Using color modes in Plone 6.0
+
+There is also the possibility to use the color modes in Plone 6.0 by manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`.
```
Comment (stevepiercy) [inline docs/classic-ui/theming/color-mode.md]: Manually include... where? `requirements.txt`? Please be explicit.

```suggestion
To use color modes in Plone 6.0, manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`, as described in {ref}`pin-the-version-of-an-add-on`.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A couple more corrections that I missed on my first pass, and addition of a `versionadded`.

I also asked a couple of questions, as a couple of things are not clear to me. Thank you!

File: docs/classic-ui/theming/color-mode.md (hunk 4 above)
Comment (toalba) [reply docs/classic-ui/theming/color-mode.md]: You right i will add it. Adding the version binding to the `requirements.txt` will do it. If working with mxdev you can add it to the mx.ini, but i think this information is to explicit right? @stevepiercy

File: docs/classic-ui/theming/color-mode.md (hunk 1 above)
Comment (toalba) [reply docs/classic-ui/theming/color-mode.md]: This is just a example on using the `data-bs-theme` attribute on a from. I don't know if the difference of `id="exampleInputEmail1"` and `id="exampldescribtioneInputEmail1"` is important, but i am new to docs so if you say, i add it.

File: docs/classic-ui/theming/color-mode.md (hunk 4 above)
Comment (stevepiercy) [reply docs/classic-ui/theming/color-mode.md]: @toalba OK, I think I understand now. I updated my suggestion with a MyST reference to https://6.docs.plone.org/install/manage-add-ons-packages.html#pin-the-version-of-an-add-on. Let's check the build or preview to make sure it renders correctly.

File: docs/classic-ui/theming/color-mode.md (hunk 1 above)
Comment (stevepiercy) [reply docs/classic-ui/theming/color-mode.md]: `exampldescribtioneInputEmail1` looks like a typographical mistake to me. That's why I asked. I think the original `exampleInputEmail1` is descriptive enough.

File: docs/classic-ui/theming/color-mode.md (hunk 4 above)
Comment (toalba) [reply docs/classic-ui/theming/color-mode.md]: ah ok, thanks

Comment (toalba) [reply docs/classic-ui/theming/color-mode.md]: The Reference is not showing in the preview, this is unknown terrain for me. can you look? @stevepiercy

File: docs/classic-ui/theming/color-mode.md [hunk 5]
```diff
@@ -149,4 +149,4 @@ See the following example.
 
 ## Using color modes in Plone 6.0
 
-There is also the possibility to use the color modes in Plone 6.0 by manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`.
+To use color modes in Plone 6.0, manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`, as described in {ref}`pin-the-version-of-an-add-on`.
```
Comment (stevepiercy) [inline docs/classic-ui/theming/color-mode.md]: My apologies, I gave you the incorrect reference. This time I looked it up in the source.

```suggestion
To use color modes in Plone 6.0, manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`, as described in {ref}`manage-add-an-add-on`.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
Let's wait for a green build, and ensure that the preview looks correct https://deploy-preview-1443--6-docs-plone-org.netlify.app/classic-ui/theming/color-mode.html

I'm OK with merging this now. Yes, it's premature, but the momentum is here, and the versioning issue has been addressed adequately, so I'll go with it. We need docs badly.

@petschki @MrTango any last thoughts?

**Review by petschki — APPROVED** [review]
Nice! Thanks @toalba !

**Comment by toalba** [comment]
Thanks as well to @lenadax , who initially developed this feature.

**Comment by rnixx** [comment]
Then it's time to drop alpha releases of mockup, staticresources and barceloneta including bs 5.3 i guess ;)

**Pull request merged**